changes/7517-token-rotation

```diff
@@ -0,0 +1,2 @@
+- Added functionality to consider device tokens as expired after one hour.
+  This change is not compatible with older versions of Fleet Desktop, If you're managing your own TUF server we recommend to increment Fleet Desktop to > v1.0.0 in addition to upgrading the server.
```

Comment:
If you're managing your own TUF server...

... or have auto-updates disabled (`fleetctl package [...] --disable-updates`)
... or have desktop+orbit channels pinned to an older version (`fleetctl package [...] --orbit-channel 1.0.0 --desktop-channel 1.1.0`)

Reply:
great, great catch, thanks. Updated